pkg/clients/clients.go

```diff
@@ -7,7 +7,7 @@ import (
 )
 
 var instances struct {
-	Etcd       *etcd.Client
+	Etcd       etcd.Client
```

Comment:
Ok, looks like etcd `Client` is an interface. 

```
type Client interface {
```

And when you change the above to a pointer, you get:

```
$ make 
go build -i -ldflags="-s -w" ./cmd/broker
# github.com/openshift/ansible-service-broker/pkg/app
pkg/app/app.go:187: etcdClient.GetVersion undefined (type *client.Client is pointer to interface, not interface)
Makefile:17: recipe for target 'broker' failed
make: *** [broker] Error 2
```

So +1 to this staying `etcd.Client`.